fdbserver/Status.actor.cpp

```diff
@@ -2836,6 +2904,7 @@ ACTOR Future<StatusReply> clusterGetStatus(
 		state std::vector<BlobWorkerInterface> blobWorkers;
 		state JsonBuilderObject qos;
 		state JsonBuilderObject data_overlay;
+		state JsonBuilderObject storage_wiggler;
```

Comment:
mix of variable naming conventions?

Reply:
I think in FDB we usually use the `snake_case` convention like the variable names (qos, data_overlay) above : )

Reply:
From the code most of the variable names are NOT SnakeCase:

		state std::map<std::string, std::vector<JsonBuilderObject>> processIssues =
		    getProcessIssuesAsMessages(workerIssues);
		state std::vector<std::pair<StorageServerInterface, EventMap>> storageServers;
		state std::vector<std::pair<TLogInterface, EventMap>> tLogs;
		state std::vector<std::pair<CommitProxyInterface, EventMap>> commitProxies;
		state std::vector<std::pair<GrvProxyInterface, EventMap>> grvProxies;
		state std::vector<BlobWorkerInterface> blobWorkers;
		state JsonBuilderObject qos;
		state JsonBuilderObject data_overlay;

Is it possible that JSONXX objects follow SnakeCase convention?

Reply:
Sry I omit some words in the comment by accident. Yeah, here I follow the name convention of the above variables but I can also rename other names.